src/main/cpp/src/utils/vid_mapper.cc

```diff
@@ -388,7 +388,9 @@ const {
         || (field_info.get_genomicsdb_type().get_num_elements_in_tuple() > 1u) //tuple has multiple elements
        )
       continue;
+    std::cout << "field " << field_info.m_name << std::endl;
```

Comment:
Can we use the genomicsdb_logger logger.debug(...) functionality instead of std::cout so these don't inadvertently creep into production? Maybe, these changed don't need to be in the PR...